compiler/rustc_resolve/src/imports.rs

```diff
@@ -991,9 +991,9 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
                     if !is_prelude
                     && let Some(max_vis) = max_vis.get()
                     && !max_vis.is_at_least(import.expect_vis(), self.tcx)
```

Comment:
The correct style includes indenting the `&&`s. But not worth changing -- we'll be getting formatting for let-chains soon (rust-lang/rustfmt#5910)

```suggestion
                        && let Some(max_vis) = max_vis.get()
                        && !max_vis.is_at_least(import.expect_vis(), self.tcx)
```